kurtosis/src/nodes/nodes.star

```diff
@@ -33,7 +33,7 @@ EXECUTION_DEFAULT_SETTINGS = {
     },
     "images": {
         "besu": "hyperledger/besu:latest",
-        "erigon": "thorax/erigon:v2.60.1",
+        "erigon": "erigontech/erigon:v2.60.1",
```

Comment:
We may want to change the docker version here, doc reference`Images before 2.60.7 are released in repository [“thorax/erigon”](https://hub.docker.com/r/thorax/erigon/tags) and will not be moved to the new repo;`. The new repo has images from 2.60.8 onwards.